src/components/auth/login/Login.jsx

```diff
@@ -0,0 +1,46 @@
+import React from "react";
+import { auth } from "../../config/firebaseConfig";
+import db from "../../config/firebaseConfig";
+import AddProducts from "../../addProductsForm/AddProducts";
+import GoogleAndFacebookLogIn from "./GoogleAndFacebookLogIn";
+
+const LogInForm = () => {
+  const [isAdmin, setIsAdmin] = React.useState(false);
+
+  const userLogout = () => {
+    auth.signOut().then(() => {
+      console.log("user logged out");
+      setIsAdmin(false);
+    });
+  };
+  const userLogin = async (e) => {
+    e.preventDefault();
+    const email = e.target[0].value;
```

Comment:
does this work? i didn't know that e.target could be used like this. also console.log in this fn needs to be cleaned up

Reply:
we did it this way instead of using ids.